content/en/apps/guides/database/muting_in_dashboards.md

```diff
@@ -0,0 +1,86 @@
+---
+title: "Contact Muting Guideline in SQL queries"
+linkTitle: "Contact Muting in SQL queries"
+weight: 
+description: >
+  How to write SQL queries excluding muted contacts correctly
+relevantLinks: >
+---
+
+When a contact gets muted, two of [many things](https://docs.communityhealthtoolkit.org/apps/reference/app-settings/transitions/#muting) happen:
+
+- The target contact and all of its descendants have a `muted` property set equal to the date they were muted
+- an entry is added to the contact's `muting_history` in sentinel's info docs
+
+When building dashboards on KlipFolio, you might need to exclude these muted contacts from analytics. An easy way to do this is to check the contact's `muted` property which when present has the date value of when the contact was muted and when absent means that the contact is not muted. This works work when you're only interested in the seeing the latest data but it gets a little bit complicated when you want to look at a contact's mute state from a certain period in the past. 
+
+For example, Say a contact was muted on February and unmuted on May; If you check the contact's mute state in March from June, you'd find that the contact would not have the muted property as it would have been removed during the `unmute` in May. This is where the `muting_history` comes in. A contact's `muting_history` contains all mute and unmute events stored in a json array. An example of the mute/unmute entries
```

Comment:
```suggestion
For example, if a contact was muted in February and unmuted in May; If you check the contact's mute state in March from June, you'd find that the contact would not have the muted property as it would have been removed during the `unmute` in May. This is where the `muting_history` comes in. A contact's `muting_history` contains all mute and unmute events stored in a JSON array. An example of the mute/unmute entries is shown below:
```